### ISSUE TITLE
Can you update the pull from  jaroslawhartman / withings-sync?

### ISSUE THREAD

**Issue by findthelorax** [maruina/docker-withings-sync#32]
TypeError: unsupported operand type(s) for /: 'NoneType' and 'float'

fix: https://github.com/jaroslawhartman/withings-sync/issues/75 TypeError during first sync by @longstone in https://github.com/jaroslawhartman/withings-sync/pull/76

Would love this to be updated if you get a chance :)

**Comment by longstone** [comment]
This doesn't solve the issue, but:

On the root repository, you will get the latest version. Also we have added documentation for docker cron there.

**Comment by findthelorax** [comment]
Hi, sorry to keep bothering you. I'm very new to this so just trying to understand. I ended up learning to push my own, all I changed was removing ARG WITHINGS_SYNC_COMMIT from the dockerfile which I though was stopping it from pulling the latest from jaroslawhartman. Is this not the root repository?

**Comment by maruina** [comment]
Hey @findthelorax,
I think that @longstone is trying to say that this is not the official docker image. You stumbled upon my docker image when I needed to build from a specific commit and I wanted it running for my own needs.

I think you will find instructions to run in docker from https://github.com/jaroslawhartman/withings-sync, but if you still want to use my image I just updated it to the latest version.
